client/.eslintrc

```diff
@@ -2,10 +2,10 @@
   "parser": "babel-eslint",
   "extends": "airbnb",
   "settings": {
-    import/resolver: "webpack"
+    "import/resolver": "webpack"
```

Comment:
@kxmbrian The configuration here does not seem to affect anything, do you know what's this was used for?